mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -580,14 +580,18 @@ def _get_v3io_frames_client(
         )
 
     @staticmethod
-    def _get_endpoint_filter(endpoint_id: Union[str, list[str]]):
+    def _get_endpoint_filter(endpoint_id: Union[str, list[str]]) -> Optional[str]:
         if isinstance(endpoint_id, str):
             return f"endpoint_id=='{endpoint_id}'"
         elif isinstance(endpoint_id, list):
             if len(endpoint_id) > V3IO_MEPS_LIMIT:
-                raise mlrun.errors.MLRunInvalidArgumentError(
-                    f"Filtering more than {V3IO_MEPS_LIMIT} model endpoints in the V3IO connector is not supported."
+                logger.warning(
+                    "The number of endpoint ids exceeds the limit, "
+                    "we have to retrieve all the model endpoints from the db.",
+                    limit=V3IO_MEPS_LIMIT,
+                    amount=len(endpoint_id),
```

Comment:
I think info severity is enough (warning severity is "scary" - and we are working within our intended design here)
```suggestion
                logger.info(
                    "The number of endpoint ids exceeds the v3io-engine filter-expression limit, "
                    "retrieving all the model endpoints from the db.",
                    limit=V3IO_MEPS_LIMIT,
                    amount=len(endpoint_id),
```